cli/utils.go

```diff
@@ -93,6 +95,24 @@ func mustGetContextConfig(cmd *cobra.Command) *viper.Viper {
 	return cmd.Context().Value(cfgContextKey).(*viper.Viper) //nolint:forcetypeassert
 }
 
+// MessageChans contains message channels that can be used to relay important information
+// after calling `cli.Start`.
+type MessageChans struct {
```

Comment:
question: Do you imagine production code use cases for this chan?

Reply:
If devs decide to use `node` in a goroutine in their project and they want to get back information from the `node`.

Reply:
This is defined in the CLI package though, which they would probably not be using if they were using `node`?

Reply:
>  Let me know if you're ok with this.

LGTM, thanks :)

> ...

Is a nice thought, although I have mixed feelings - mostly I think because I see `node` as a simplified wrapper around the stuff that *does* allow the more complex customisation - if they want to play around with different event busses they should probably not be using the `node` package.